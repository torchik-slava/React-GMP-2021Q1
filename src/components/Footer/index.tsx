import React from "react";
import Logo from "../Logo";
import styles from "./styles.module.scss";

const Footer = (): React.ReactElement => {
  return (
    <footer className={styles.footer}>
      <Logo />
    </footer>
  );
};

export default Footer;
