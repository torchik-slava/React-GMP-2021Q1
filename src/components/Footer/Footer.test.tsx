import React from "react";
import { render } from "@testing-library/react";
import Footer from "./Footer";

describe("Footer", () => {
  it("renders Footer correctly", () => {
    const { asFragment } = render(<Footer />);
    expect(asFragment()).toMatchSnapshot();
  });
});
