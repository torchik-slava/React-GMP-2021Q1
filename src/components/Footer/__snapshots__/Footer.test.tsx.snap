// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders Footer correctly 1`] = `
<DocumentFragment>
  <footer
    class="footer"
  >
    <span
      class="logo"
    >
      <span
        class="bold"
      >
        netflix
      </span>
      roulette
    </span>
  </footer>
</DocumentFragment>
`;
